cmd/lncli/cmd_open_channel.go

```diff
@@ -191,6 +196,7 @@ func openChannel(ctx *cli.Context) error {
 		MinConfs:         minConfs,
 		SpendUnconfirmed: minConfs == 0,
 		CloseAddress:     ctx.String("close_address"),
+		MaxValueInFlight: ctx.Uint64("remote_max_value_in_flight_msat"),
```

Comment:
The `lnrpc` commit (9842639e03b3e510dcf385c92c3b28d5b8121ffb) should come before this one (56dd35e3424d7ae5da512390febe1720cb524a34).